sources/packages/api/src/services/student-scholastic-standings/student-scholastic-standings.service.ts

```diff
@@ -18,8 +20,18 @@ import {
 } from "../application/application.service";
 import { ScholasticStanding } from "./student-scholastic-standings.model";
 import { StudentAssessmentService } from "../student-assessment/student-assessment.service";
+import { StudentRestrictionService } from "../restriction/student-restriction.service";
+import { RestrictionCode } from "../restriction/constants";
 import { APPLICATION_CHANGE_NOT_ELIGIBLE } from "../../constants";
 
+// When student select "Student did not complete program", then "scholasticStanding" value is
+const SCHOLASTIC_STANDING_STUDENT_DID_NOT_COMPLETE_PROGRAM =
```

Comment:
We can have a `constants.ts` in this service folder to save these ones. There are other services following it.